Pokemon Tabletop Adventure/Supporting Files/JSON/abilites.json

```diff
@@ -0,0 +1,469 @@
+            "keyword": [
+                
+            ],
+            "effect": "A pokemon may not use the Moves Explosion or Self-Destruct, nor activate the Ability Aftermath, within 20 meters of the user."
+        },
+        {
+            "name": "Dark Art",
+            "type": "Trigger",
+            "keyword": [
+                "Last Chance"
+            ],
+            "effect": "Dark"
+        },
+        {
+            "name": "Defiant",
+            "type": "Static",
+            "keyword": [
+                
+            ],
+            "effect": "Whenever the user has a Combat Stage lowered by an external force, its Attack is raised by 2 Combat Stages for each stage the other was lowered."
+        },
+        {
+            "name": "Dry Skin",
+            "type": "Static",
+            "keyword": [
+                
+            ],
+            "effect": "The user loses 1/8th of its maximum HP at the beginning of each round while in Sunny Weather, and heals the same amount in Rainy Weather. Fire type Moves deal damage as if the attacker were 1 Combat Stage higher in the relevant stat, and Water type Moves heal the user for the amount of damage they would cause if the user was Resistant to them."
+        },
+        {
+            "name": "Early Bird",
+            "type": "Staic",
+            "keyword": [
+                
+            ],
+            "effect": "When making Sleep Checks, the counter starts at 11."
+        },
+        {
+            "name": "Fae Force",
+            "type": "Trigger",
+            "keyword": [
+                "Last Chance"
+            ],
+            "effect": "Fairy"
+        },
+        {
+            "name": "Flame Body",
+            "type": "Trigger",
+            "keyword": [
+                "Daily"
+            ],
+            "effect": "When a foe hits the user with a Melee move, they become Burned after damage is dealt."
+        },
+        {
+            "name": "Flash Fire",
+            "type": "Static",
+            "keyword": [
+                
+            ],
+            "effect": "The user does not take damage from Fire moves and may not be Frozen. If hit by a Fire move, the user adds +1/2 STAB to the next Fire Type move they use in the current encounter."
+        },
+        {
+            "name": "Flower Gift",
+            "type": "Trigger",
+            "keyword": [
+                "Burst",
+                "Hourly"
+            ],
+            "effect": "In Sunny Weather, Flower Gift creates a 4-meter burst. The user and all of their allies in Sunny Weather have their Attack and Special Defense raised by 2 Combat Stages."
+        },
+        {
+            "name": "Focus",
+            "type": "Trigger",
+            "keyword": [
+                "Last Chance"
+            ],
+            "effect": "Fighting"
+        },
+        {
+            "name": "Frisk",
+            "type": "Cast",
+            "keyword": [
+                "Hourly"
```

Comment:
@d9rad 
These kinds of abilities are listed in the rulebook as "Cast - Hourly", but that raises questions regarding how to enumerate types:
Do we want to have "Daily Cast", "Hourly Cast", "Trigger", "Static", or just keep the time-frames as keywords like above?